### PR TITLE
Dee jm customizations 01042022

### PR DESCRIPTION
**Pull request by dee-luo**
Uploaded Jamaica form definition customizations

### CODE REVIEW

**Review by stephenhand — APPROVED** [review]

File: plugin-hrm-form/src/formDefinitions/jm-v1/caseForms/NoteForm.json [hunk 1]
```diff
@@ -0,0 +1 @@
+[]
```
Comment (murilovmachado) [inline plugin-hrm-form/src/formDefinitions/jm-v1/caseForms/NoteForm.json:1]: Won't SafeSpot have Add Note/View Note?

**Review by murilovmachado — COMMENTED** [review]
Looks good, just one question on `NoteForm.json`

**Review by murilovmachado — APPROVED** [review]

File: plugin-hrm-form/src/formDefinitions/jm-v1/caseForms/NoteForm.json (hunk 1 above)
Comment (dee-luo) [reply plugin-hrm-form/src/formDefinitions/jm-v1/caseForms/NoteForm.json]: Oh good catch thank you @murilovmachado! Updated.

File: plugin-hrm-form/src/formDefinitions/jm-v1/CaseStatus.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (nickhurlburt) [inline plugin-hrm-form/src/formDefinitions/jm-v1/CaseStatus.json:1]: Can we remove this whitespace?

File: plugin-hrm-form/src/formDefinitions/jm-v1/HelplineInformation.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+
+
```
Comment (nickhurlburt) [inline plugin-hrm-form/src/formDefinitions/jm-v1/HelplineInformation.json:2]: whitespace

File: plugin-hrm-form/src/formDefinitions/jm-v1/caseForms/ReferralForm.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        { "value": "Child Protection and Family Services Agency (CPFSA)", "label": "Child Protection and Family Services Agency (CPFSA)" },
+        { "value": "National Children's Registry", "label": "National Children's Registry" },
+        { "value": "Office of the Children's Advocate", "label": "Office of the Children's Advocate" },
+        { "value": "Police", "label": "Police" },
+        { "value": "Other", "label": "Other" }
+      ],
+      "required": { "value": true, "message": "RequiredFieldError" }
+    },
+    {
+        "name": "reasonforReferral",
```
Comment (nickhurlburt) [inline plugin-hrm-form/src/formDefinitions/jm-v1/caseForms/ReferralForm.json:25]: Maybe this was intentional, but do you want to camel case this as `reasonForReferral`?

**Review by nickhurlburt — APPROVED** [review]
Noted a couple small things, otherwise good!

**Pull request merged**